CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 1.9.0
+- Support AVM 1.0
+  - Update TEAL langspec to v5
+  - Add function to get app address
+  - Regenerate REST API types
+  - Support new cucumber tests
+  - Enable new cucumber tests
+- Allow non-base64 metadata hash and force 32 bytes
+- mark contract templates as @Deprecated
+- Added pretty print option to Object Mapper instance
+- Delete generator since it has it's own repo now
```

Comment:
I don't think this needs to be called out in the changelog, it doesn't effect the release

Reply:
removed 👍